### PR TITLE
[feat] Sets up basic controller/view structures for app navigation

### PR DESCRIPTION
**Pull request by marcoviappiani**
- app routing structure with dynamic nested states and views
- basic client navigation
- basic structure of controllers and views with nearly empty files
- adjusted tests (but no major test introduced)

### CODE REVIEW

File: modules/core/client/views/core.client.schedule.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>Schedule section</h1>
+<div>
+	Schedule content
+</div>
+
```
Comment (codydaig) [inline modules/core/client/views/core.client.schedule.html]: Extra New Line

File: modules/core/client/views/core.client.blog.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<h1>Blog section</h1>
+<div>
+	Blog content
+</div>
+
```
Comment (codydaig) [inline modules/core/client/views/core.client.blog.html]: Extra New Line

File: modules/core/client/views/core.client.app.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<div ui-view name="navigation-menu"></div>
+<div ui-view name="main-content"></div>
```
Comment (codydaig) [inline modules/core/client/views/core.client.app.html]: Need a New Line

File: modules/core/client/views/core.client.about.html [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<h1>About section</h1>
+<div>
+	About content
+</div>
+
```
Comment (codydaig) [inline modules/core/client/views/core.client.about.html]: Excess New Line

File: modules/core/client/controllers/core.client.scheduleController.js [hunk 5]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+angular.module('tropicalbs.schedule',[])
+	.controller('ScheduleController', function($scope) {
+	});
+
+
```
Comment (codydaig) [inline modules/core/client/controllers/core.client.scheduleController.js]: Excess New Lines

File: modules/core/client/controllers/core.client.blogController.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+angular.module('tropicalbs.blog',[])
+	.controller('BlogController', function($scope) {
+	});
+
+
```
Comment (codydaig) [inline modules/core/client/controllers/core.client.blogController.js]: Excess New Lines

File: modules/core/client/controllers/core.client.appController.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+angular.module('tropicalbs.app',[])
+	.controller('AppController', function($scope) {
+	});
+
+
```
Comment (codydaig) [inline modules/core/client/controllers/core.client.appController.js]: Excess New Lines

File: modules/core/client/controllers/core.client.adminController.js [hunk 8]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+angular.module('tropicalbs.admin',[])
+	.controller('AdminController', function($scope) {
+		$scope.content = 'this is some controller generated content';
+	});
+
```
Comment (codydaig) [inline modules/core/client/controllers/core.client.adminController.js]: Excess New Line

File: modules/core/client/controllers/core.client.aboutController.js [hunk 9]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+angular.module('tropicalbs.about',[])
+	.controller('AboutController', function($scope) {
+	});
+
+
```
Comment (codydaig) [inline modules/core/client/controllers/core.client.aboutController.js]: Excess New Lines

File: modules/core/client/views/core.client.app.html (hunk 3 above)
Comment (marcoviappiani) [inline modules/core/client/views/core.client.app.html]: thanks

**Comment by codydaig** [comment]
LGTM

**Pull request merged**